### ISSUE TITLE
KeyError: 'graphql'

### ISSUE THREAD

**Issue by anzenchitai** [InstaPy/InstaPy#6199]
`Traceback (most recent call last):
  File "quickstart.py", line 16, in <module>
    with smart_run(session):
  File "C:\Users\rwds\AppData\Local\Programs\Python\Python37\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\rwds\Desktop\a\instapy\util.py", line 1931, in smart_run
    session.login()
  File "C:\Users\rwds\Desktop\a\instapy\instapy.py", line 470, in login
    self.followed_by = log_follower_num(self.browser, self.username, self.logfolder)
  File "C:\Users\rwds\Desktop\a\instapy\print_log_writer.py", line 20, in log_follower_num
    followed_by = getUserData("graphql.user.edge_followed_by.count", browser)
  File "C:\Users\rwds\Desktop\a\instapy\util.py", line 469, in getUserData
    data = data[subobject]
KeyError: 'graphql'`

This happens right after Instapy tries to save account progress. 
`Unable to save account progress, skipping data update
        b"'graphql'"`

**Comment by manzke** [comment]
Same for mee 0.6.3 MacOS 

 Unable to save account progress, skipping data update 
	b'Message: TypeError: window._sharedData.entry_data.ProfilePage[0].graphql is undefined\n'

**Comment by fipooo** [comment]
Same for mee.
```
Unable to save account progress, skipping data update 
	b"'ProfilePage'"
Sessional Live Report:
	|> No any statistics to show
[Session lasted 2.44 minutes]
OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2021-06-01 12:08:02] [xyz@gmail.com]  Session ended!
oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo
Traceback (most recent call last):
  File "/Users/macbook/Documents/instapy_bot.py", line 47, in <module>
    with smart_run(session):
  File "/usr/local/Cellar/python@3.7/3.7.8_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/usr/local/lib/python3.7/site-packages/instapy/util.py", line 1931, in smart_run
    session.login()
  File "/usr/local/lib/python3.7/site-packages/instapy/instapy.py", line 470, in login
    self.followed_by = log_follower_num(self.browser, self.username, self.logfolder)
  File "/usr/local/lib/python3.7/site-packages/instapy/print_log_writer.py", line 20, in log_follower_num
    followed_by = getUserData("graphql.user.edge_followed_by.count", browser)
  File "/usr/local/lib/python3.7/site-packages/instapy/util.py", line 462, in getUserData
    data = shared_data["entry_data"]["ProfilePage"][0]
KeyError: 'ProfilePage'
```

**Comment by schealex** [comment]
possibly related to #6195 and fixed by PR #6195. Can you please check if that PR fixes your issue too and leave feedback?
